samples/ChangeFeedMigrationTool/Program.cs

```diff
@@ -0,0 +1,144 @@
+﻿//--------------------------------------------------------------------------------- 
+// <copyright file="Program.cs" company="Microsoft">
+// Microsoft (R)  Azure SDK 
+// Software Development Kit 
+//  
+// Copyright (c) Microsoft Corporation. All rights reserved.   
+// 
+// THIS CODE AND INFORMATION ARE PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,  
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES  
+// OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.  
+// </copyright>
+//---------------------------------------------------------------------------------
+
+namespace ChangeFeedMigrationSample
+{
+    using Microsoft.Azure.Documents.ChangeFeedProcessor;
+    using Microsoft.Azure.Documents;
+    using Microsoft.Azure.Documents.Client;
+    using System;
+    using System.Threading.Tasks;
+
+    /// ------------------------------------------------------------------------------------------------
+    /// This sample demonstrates using change processor library to read changes from source collection 
+    /// to destination collection 
+    /// ------------------------------------------------------------------------------------------------
+
+
+    class Program
+    {
+        // Modify to connect to your own subscription and database for monitored collection 
+        private string monitoredUri = "https://URI";
```

Comment:
Since we have app.config, why don't we read these from App.config? This way the code doesn't have to be recompiled when parameters change.